packages/terra-clinical-action-header/package.json

```diff
@@ -21,9 +21,6 @@
     "url": "https://github.com/cerner/terra-clinical/issues"
   },
   "homepage": "https://github.com/cerner/terra-clinical#readme",
-  "devDependencies": {
-    "terra-toolkit": "^0.x"
-  },
```

Comment:
We should definitely update the generator.  We can't do it in terra-core.  It's a chicken and an egg kind of thing unfortunately.  Can't depend on something at the root level if it's a package